packages/jsii/lib/assembler.ts

```diff
@@ -482,6 +482,15 @@ export class Assembler implements Emitter {
             LOG.trace(`Processing enum: ${colors.gray(namespace.join('.'))}.${colors.cyan(type.symbol.name)}`);
         }
 
+        const decl = type.symbol.valueDeclaration;
+        const flags = ts.getCombinedModifierFlags(decl);
+        // tslint:disable-next-line:no-bitwise
+        if (flags & ts.ModifierFlags.Const) {
+            this._diagnostic(decl,
+                             ts.DiagnosticCategory.Error,
+                            `Exported enum cannot be declared 'const'`);
+        }
```

Comment:
Should we return from there? So we don't include it in the `.jsii` assembly we'd create?

Reply:
No.